src/microbe_stage/MicrobeSystem.cs

```diff
@@ -23,7 +23,8 @@ public void Process(float delta)
         if (delta <= 0)
             return;
 
-        var microbes = worldRoot.GetTree().GetNodesInGroup(Constants.AI_TAG_MICROBE).Cast<Microbe>().ToArray();
+        var microbes = worldRoot.GetTree().GetNodesInGroup(Constants.PROCESSABLE_MICROBE_GROUP).Cast<Microbe>()
```

Comment:
Ah I see now why you added the new group... though couldn't this be handled so that this just then filters out the microbes that are in invalid state for processing, or am I missing something?

Reply:
I don't think microbes currently has any invalid state for processing, they still need to be processed to run some functions even when being engulfed.

Reply:
So why is this separate group then needed at all?

Reply:
Because `AI_TAG_MICROBE` is used by the hover panel to get all microbes, and when an individual is engulfed we don't want it to be shown in there, so its `AI_TAG_MICROBE` group is cleared along with others, and due to this we require another group for MicrobeSystem to work and this group is the only one that's not cleared when some microbe is engulfed.

Reply:
Okay, that I can understand. Then all's that left is that the constant should be renamed. Because "processing" is way too linked to the process system. So I think it should be called  "NORMAL_MICROBE_GROUP" or "MICROBE_GROUP" or maybe something else but I don't have any thinking power left to come up with anything better. Also this *definitely* needs to be commented in Constants as t owhat's the difference.

Reply:
Indeed, I'll call it RUNNABLE_MICROBE_GROUP, I think it sounds a bit accurate to what it does.